libobs/util/base.c

```diff
@@ -64,6 +64,9 @@ OBS_NORETURN static void def_crash_handler(const char *format, va_list args,
 }
 
 static log_handler_t log_handler = def_log_handler;
+#if !defined(_MSC_VER) && !defined(SWIG)
+OBS_NORETURN
```

Comment:
@notr1ch Correct me if I'm wrong, but I guess this is not good, because this would disable the NORETURN attribute when compiling with MSVC.